docs/Processes.md

```diff
@@ -1435,3 +1435,200 @@ and are under the `.ctp` namespace.
+    3  .dqc.constructcheck quote,table                              rdb         rdb1         2020.03.19D20:08:00.378141000 2020.03.19D20:08:00.398781000 1      "quote table exists"                             complete  scheduled
+    4  .dqc.constructcheck date,variable,comp(hdb1,0)               hdb         hdb2         2020.03.19D20:08:00.379574000 2020.03.19D20:08:00.404501000 1      "hdb1  | match hdb2"                             complete  scheduled
+```
+
+The columns of results table provide information about the checks performed:
+
+**id** - The id of the check after being loaded into the process. In the
+sample table above, `.dqc.construckcheck` with the id 3 that checks whether
+quote table exists and `.dqc.construckcheck` with the id 4 that checks
+whether data in hdb1 matches hdb2 had two results showing as they have been
+ran twice within that time period, but the parameters of the checks have not
+been changed. The id column is also useful for manually running checks.
+
+**funct** - The check function that was performed.
+
+**params** - The variables that were input while the check function was 
+performed.
+
+**procs** - The process(es) type that the function was performed on.
+
+**procschk** - The specific process(es) that the function was performed on.
+
+**starttime** - When the function was started.
+
+**endtime** - When the function stopped running.
+
+**result** - Returns a boolean value. if the **result** is `1`, then the
+function was ran successfully, and no data anomaly was found. If the
+**result** is `0`, then the function may not have been run successfully,
+or the data quality may be corrupted.
+
+**descp** - A string description describing the result of the check 
+function.
+
+**chkstatus** - Could display either `complete` or `failed`. When the
+check function is successfully ran, whether the **result** column is 0 or 1,
+**chkstatus** would be `complete`. However, if there was an error that caused
+the check to not run normally(Ex: variables being a wrong type), `failed` 
+would be displayed instead.
+
+**chkruntype** - Could display either `scheduled` or `manual`, meaning
+either the check was ran as scheduled from the configtable, or it was forced
+to run manually at a certain time.
+
+Data Quality Engine (DQE)
+-------
+
+The Data Quality Engine (DQE) process that stores daily statistics about
+other TorQ processes. It is a separate process from the Data Quality
+Checker because the Engine doe not run checks. The Engine and the Checker
+do not directly engage with each other. The daily statistics of other TorQ
+processes are saved to Data Quality Engine Database(DQEDB), which would be
+used by the Checker to perform advanced checks. For example, the engine
+could be used to track the percentage change of records in a table from day 
```

Comment:
I think this needss worded better as the engine is not being used to track the percentage change at this point right?  The engine is collecting count and the checker does the percentage change check